doc/Type/Seq.pod6

```diff
@@ -78,23 +80,26 @@ Returns the underlying iterator, and marks the invocant as consumed.
 If called on an already consumed sequence, throws an error of type
 L<X::Seq::Consumed|/type/X::Seq::Consumed>.
 
-=head2 method elems
-
-    method elems(Seq:D:)
-
-If the caller C<Seq> is not lazy, consumes and caches its values,
-returning their length. Otherwise, throws an error of type
-L<X::Cannot::Lazy|/type/X::Cannot::Lazy>.
-
 =head2 method is-lazy
 
     method is-lazy(Seq:D: --> Bool:D)
 
-Returns C<True> if the sequence is lazy and potentially infinite, and C<False>
-otherwise.
-If called on an already consumed sequence, throws an error of type
+Returns C<True> if and only if the underlying iterator guarantees lazy
+evaluation and has not been completely evaluated. If called on an
+already consumed sequence, throws an error of type
 L<X::Seq::Consumed|/type/X::Seq::Consumed>.
 
+=head2 method elems
```

Comment:
Why was this moved? Did you want the is-lazy method to be as far up as possible?

Reply:
The description of `.elems` refers to laziness, so I thought it better to document `.is-lazy` first. Also, I thought it better to document `.is-lazy` near the property that determines if a sequence is lazy.

Reply:
These pages are not really intended to read as a narrative, but more to check individual methods, mainly on their own pages. But it's OK, I was just curious.